src/logic/notifications/notificationTypes.js

```diff
@@ -10,15 +10,15 @@ export const INFO = 'info'
 const shortDuration = 5000
 const longDuration = 10000
 
-export type Variant = SUCCESS | ERROR | WARNING | INFO
+export type Variant = 'success' | 'error' | 'warning' | 'info'
```

Comment:
because flow doesn't support this syntax.
```
A name can be used as a type only if it refers to a type definition, an interface definition, or a class definition. To get the type of a non-class value, use `typeof`.Flow(InferError)
```